packages/titus-db-manager/lib/routes.js

```diff
@@ -11,13 +11,35 @@ const Migrate = require('../migrate')
 const Truncate = require('../truncate')
 const { version } = require('../package')
 
+const inputSchema = {
+  oneOf: [
+    { type: 'null' },
+    {
+      description: 'Optional schema to process',
+      type: 'object',
+      properties: {
+        schema: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 64
+        }
+      }
+    }
+  ]
+}
+
 const config = require('./config')
 async function dbRoutes(server) {
+  server.decorateRequest('pgSchema', function pgSchema() {
+    return (this.body && this.body.schema) || 'public'
```

Comment:
why do we need this decorator? why not put the default value in the schema definition instead?

Reply:
we can't use the `default` value here because we are accepting a `null` body to support the old interface.
If we remove the `null` body, fastify triggers an error when a client does an HTTP POST without an empty JSON body `{}`.